<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>buycarwin</class>
 <widget class="QWidget" name="buycarwin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>891</width>
     <height>531</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>889</width>
      <height>529</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#scrollAreaContents{
    background:transparent;
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="allPay_lab">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>650</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
   color: #ffffff; /* 设置文本颜色为白色 */
   font-size: 20px; /* 设置字体大小 */
   padding: 20px; /* 设置内边距 */
}
</string>
   </property>
   <property name="text">
    <string>总额：XXX元</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buy_ptn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>650</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   background-color: #4CAF50; /* 设置背景颜色为绿色 */
   color: #ffffff; /* 设置文字颜色为白色 */
   border: none; /* 去除边框 */
   border-radius: 5px; /* 设置边框为5像素的圆角 */
}
   /* 悬停样式 */
   QPushButton:hover {
       background-color: #45a049; /* 设置悬停时的背景颜色 */
   }

   /* 按下样式 */
   QPushButton:pressed {
       background-color: #3e8e41; /* 设置按下时的背景颜色 */
   }
</string>
   </property>
   <property name="text">
    <string>购买</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_ptn">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>650</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { 
   background-color: #ffcc00; /* 设置背景颜色为黄色 */
   color: #ffffff; /* 设置文字颜色为白色 */
   border: none; /* 去除边框 */
   border-radius: 5px; /* 设置边框为5像素的圆角 */
}
   /* 悬停样式 */
   QPushButton:hover {
       background-color: #e6b800; /* 设置悬停时的背景颜色 */
   }

   /* 按下样式 */
   QPushButton:pressed {
       background-color: #cca300; /* 设置按下时的背景颜色 */
   }</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>0</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>购物车</string>
   </property>
  </widget>
  <widget class="QLabel" name="buyCarIcon_lab">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ICON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteSelect_ptn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>650</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   background-color: #ff4d4d; /* 设置背景颜色为红色 */
   color: #ffffff; /* 设置文字颜色为白色 */
   border: none; /* 去除边框 */
   border-radius: 5px; /* 设置边框为5像素的圆角 */
}
   /* 悬停样式 */
   QPushButton:hover {
       background-color: #e64040; /* 设置悬停时的背景颜色 */
   }

   /* 按下样式 */
   QPushButton:pressed {
       background-color: #cc3333; /* 设置按下时的背景颜色 */
   }
</string>
   </property>
   <property name="text">
    <string>删除选中</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="allSelect_ckbox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>650</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
    background-color: #f0f0f0; /* 设置背景颜色为淡灰色 */
	border-radius: 5px; /* 设置边框为5像素的圆角 */
	padding: 3px; /* 设置内边距 */
}
    /* 悬停样式 */
    QCheckBox:hover {
        background-color: #e0e0e0; /* 设置悬停时的背景颜色 */
    }

    /* 按下样式 */
    QCheckBox:pressed {
        background-color: #c0c0c0; /* 设置按下时的背景颜色 */
    }
</string>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
